src/components/molecules/Menu/index.js

```diff
@@ -0,0 +1,46 @@
+import React from "react";
+import styled from "styled-components";
+import { Link } from "react-router-dom";
+
+const StyledMenu = styled.div`
```

Comment:
I don't really like the approach of creating wrapper divs and inside css selectors of everything. Create little components :)